lib/lotus/controller/configuration.rb

```diff
@@ -418,35 +418,92 @@ def format(hash)
       # an argument, it will set the corresponding instance variable. When
       # called without, it will return the already set value, or the default.
       #
-      # @overload default_format(format)
+      # @overload default_request_format(format)
       #   Sets the given value
       #   @param format [#to_sym] the symbol format
       #   @raise [TypeError] if it cannot be coerced to a symbol
       #
-      # @overload default_format
+      # @overload default_request_format
       #   Gets the value
       #   @return [Symbol,nil]
       #
-      # @since 0.2.0
       #
       # @see Lotus::Action::Mime
       #
       # @example Getting the value
       #   require 'lotus/controller'
       #
-      #   Lotus::Controller.configuration.default_format # => nil
+      #   Lotus::Controller.configuration.default_request_format # => nil
       #
       # @example Setting the value
       #   require 'lotus/controller'
       #
       #   Lotus::Controller.configure do
-      #     default_format :html
+      #     default_request_format :html
       #   end
+      #
+      # @since 0.5.0 TODO: Confirm?
+      def default_request_format(format = nil)
+        if format
+          @default_request_format = Utils::Kernel.Symbol(format)
+        else
+          @default_request_format
+        end
+      end
+
+      # Set a format as default fallback for all the requests without a strict
+      # requirement for the mime type.
+      #
+      # @since 0.2.0
+      #
+      # @deprecated Use {#default_request_format} instead.
       def default_format(format = nil)
+        Lotus::Utils::Deprecation.new('default_format is deprecated, please use default_request_format')
```

Comment:
@theocodes We probably need a test case for this deprecation. You can use Minitest `capture_io`.

```ruby
_, err = capture_io do
  Lotus::Controller.configure do
    default_format :png
  end
end

err.chomp.must_equal "..."
```